Yona/Podfile

```diff
@@ -6,10 +6,11 @@ use_frameworks!
 target 'Yona' do
    
    pod 'R.swift'
-   #pod 'HockeySDK'
    pod 'Google/Analytics'
    pod 'Kingfisher', '~> 4.0'
    pod 'IQKeyboardManagerSwift'
+   pod 'Fabric'
```

Comment:
Is Fabric a dependency of Crashlytics?

Reply:
Yes, as Crashlytics kit is part of Fabric.
https://fabric.io/kits